ros2_batch_job/packaging.py

```diff
@@ -73,10 +73,10 @@ def build_and_package(args, job):
             '--only-packages', 'ros1_bridge',
         ] + (['--isolated'] if args.isolated else []) +
             (
-                ['--cmake-args', '-DCMAKE_BUILD_TYPE=' + args.cmake_build_type]
+                ['--cmake-args', '-DCMAKE_BUILD_TYPE=' + args.cmake_build_type + ' -- ']
                 if args.cmake_build_type else []
             ) + [
-            '--make-flags', '-j1'
+            '--make-flags', '-j1', ' -- '
```

Comment:
If I understand well that means that we should never add trailing whites spaces along end delimiters right? If I did it is because I copied what was done in https://github.com/ros2/ci/blob/master/ros2_batch_job/__main__.py#L230-L235, I guess I should remove those too then?